lib/resilient.js

```diff
@@ -184,10 +192,17 @@ Resilient.prototype.addFailStrategy = function (strategy) {
   this._failStrategies.add(strategy)
   return this
 }
-
 ;['get', 'post', 'put', 'del', 'delete', 'head', 'patch']
   .forEach(function defineMethodProxy (verb) {
-    Resilient.prototype[verb] = function (path, options, cb) {
-      return this._client[verb](path, options, cb)
+    Resilient.prototype[verb] = function (path, options) {
```

Comment:
I would like to keep interface consistency supporting both callback and promise patterns. It should be straighforward doing it.